lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -105,6 +105,7 @@ def ignore_methods?
         def filename_ends_with?(glob)
           filename =
             RuboCop::PathUtil.relative_path(processed_source.buffer.name)
+              .gsub('../', '')
```

Comment:
I was hoping to find a built-in command for doing this. I am not sure if this works on Windows, for example.

Reply:
Hi @bquorning ,

thank you for you feedback. I was not able to get my example files working with a specific gem path in the Gemfile in windows. I dont understand that, but i was able to have x.rb which shows up that the path is the same on windows:

```ruby
require 'rubocop'
p RuboCop::PathUtil.relative_path('c:\ruby\project2\spec\lib\blub_spec.rb')
```

![grafik](https://user-images.githubusercontent.com/5445062/83530835-92c76680-a4ec-11ea-8bcd-5b435b1cce58.png)

On my research on stackoverflow someone wrote this:

`One thing to note. Ruby uses a "/" for file separator on all platforms, including Windows, so you don't actually need use different code for joining things together on different platforms. "C:/tmp/1.text" should work fine.` (https://stackoverflow.com/a/597500/3005815)

So maybe we don't need a different solution for windows? Alternatively we could think about to use this static variable: `File::SEPARATOR`. 

What do you think?

Reply:
It is _probably_ fine. I know there is both `File::SEPARATOR`, `File::ALT_SEPARATOR` and `File::Separator` to take into account.

Let’s just merge it, and if anyone files a bug we’ll take if from there.

Could you just rebase the branch, then I think it’s ready to merge.

Reply:
Thank you !!! 👍 check